Agreements/External/AAlow_E.txt

```diff
@@ -1,2 +1,11 @@
+FIR_COPX:EHAM:*:ABNED:*:*:EHAA low:London AC Clacton:24000:*:ABNED
+FIR_COPX:EHAM:*:MIMVA:*:*:EHAA low:London AC Clacton:24000:*:MIMVA
+FIR_COPX:EHAM:*:KOLAG:*:*:EHAA low:London AC Clacton:24000:*:KOLAG
+
+FIR_COPX:*:*:IBNOS:*:*:EHAA low:London AC Clacton:24000:*:IBNOS
```

Comment:
IBNOS or ABNED?

Reply:
"Departing Aircraft from EHAM via ABNED
Departing Aircraft from EHAM will be transferred climbing FL240 to be level before ABNED. Traffic is released for climb by Amsterdam ACC on transfer, but coordination with Maastricht Delta is required in order to climb above FL240 before the FIR boundary. Note this route should not be used by traffic routing between the EHAM and EGSS/EGGW/EGSC.

Departing Aircraft from Amsterdam FIR via IBNOS
Departing Aircraft from Amsterdam FIR (except EHAM) will be transferred climbing FL240 to be level before IBNOS. Traffic is released for climb by Amsterdam ACC on transfer, but coordination with Maastricht Delta is required in order to climb above FL240 before the FIR boundary. Note this route should not be used by traffic routing between the Amsterdam FIR and EGSS/EGGW/EGSC/EGSH."

IBNOS EHAM only?

Reply:
I think it should be ABNED, climbing 240 and must be level before IBNOS in order to (RW) stay in S13 (FL245+) and avoid S14 (215-245)

It doesnt make sense for EHAA to have tfc between ABNED and IBNOS between 215 and 240, since this is delegated to us in this area.